style.css

```diff
@@ -0,0 +1,40 @@
+@import "./utils.css";
+
+body {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+body * {
+    box-sizing: inherit;
+}
+
+img {
+    width: 100%;
+}
+
+.gallery-column {
+    flex-basis: 31%;
+}
+
+@media screen and (max-width:980px) {
+    .hidden1 {
+        display: none;
+    }
+
+    .gallery-column {
+        flex-basis: 45%;
+    }
+}
+
+@media screen and (max-width:600px) {
+
+    .hidden1,
+    .hidden2 {
+        display: none;
+    }
+
+    .gallery-column {
+        flex-basis: 90%;
+    }
```

Comment:
Let's refactor to select them using `nth-child`, thus no need for the `hidden1` or `hidden2` classes.